packages/downloader/download.py

```
@@ -0,0 +1,126 @@
+    "youtube_include_dash_manifest": True,
+    "prefer_insecure": True,
+    "keepvideo": False,
+    "extract_flat": False,
+    "extractor_args": {
+        "youtube": {
+            "player_client": ["web"],
+        }
+    },
+}
+
+def progress_hook(data):
```

Ruff (0.12.2)

Missing return type annotation for public function `progress_hook`

Add return type annotation: `None`

(ANN201)